services/blockchain-indexer/shared/dataService/accounts.js

```diff
@@ -29,7 +29,7 @@ const getAccounts = async params => {
 	const { status, ...remainingParams } = params;
 	let response;
 	if (status) {
-		// Include delegate info in all accounts requests unless explicitly stated
+		// Include validator info in all accounts requests unless explicitly stated
```

Comment:
Out of the context for this PR but we can make a note to remove this file completely as well as all existence of `getAccounts` method from code.

Reply:
Updating issue https://github.com/LiskHQ/lisk-service/issues/1253 to take care of this 👍